app/templates/admin/settings/billing.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="ui basic {{if isLoading 'loading'}} segment">
+  {{forms/admin/settings/billing timezones=timezones  adminUser=adminUser model=model save='updateInvoiceModel'}}
```

Comment:
change `save='updateInvoiceModel'` to use closure actions

Reply:
@niranjan94 Done!